8.1/Dockerfile.fedora

```
@@ -0,0 +1,80 @@
+FROM quay.io/fedora/s2i-base:36
```

Only nitpick. Can we used s2i-base:37? 
```suggestion
FROM quay.io/fedora/s2i-base:37
```

---

F36 is going to be EOL soon.